aws/app/lambda/submission/submission.js

```diff
@@ -15,8 +15,11 @@ exports.handler = async function (event) {
   try {
     const formData = event;
     const submissionID = uuid.v4();
+    // add timestamp 
+    const editFormData = typeof formData === "string" ? JSON.parse(formData) : formData;
+    editFormData['timestamp'] = (new Date()).getTime();
```

Comment:
This is probably better suited as an ISO string. We also need to modify the reliability queue to also send this through in the email

Reply:
also It would be helpful to have this as a separate field in the DynamoDB table which would allow us to further create indexes if needed

Reply:
I now agree with the fact that it would be helpful for indexes. If we want to move forward with the second solution I proposed for https://github.com/cds-snc/platform-forms-client/issues/242 then it would be required.

Reply:
The reason why it was implemented this way was to avoid any impacts on reliability queue and also not having to pass it around etc... 
Although the need for indexing  may come along , as for now we only use it as for additional information on a given submission. 
So i guess these new requirements will definitely extend the scope of this PR.